scripts/aws-cloudhsm.ps1

```diff
@@ -0,0 +1,22 @@
+param ([string] $workdir,
+       [string] $configfiles)
+
+Write-Host "Installing AWSCloudHSMClient"
+
+. C:\Windows\Temp\scripts\ps_support.ps1
+
+# Note: uses SDK v3 for import-key.exe
+wget https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/Windows/AWSCloudHSMClient-latest.msi -Outfile C:\AWSCloudHSMClient-latest.msi
```

Comment:
Invoke-Webrequest would look less odd, but that's a matter of taste :smile:.

Reply:
Yeah, honestly just copied the example from AWS documentation. But since wget is just an alias it makes sense to replace it.